log/logClient.go

```diff
@@ -204,9 +218,62 @@ func (fd *Feeder) WatchAlerts(logPath string, jsonFormat bool) error {
 			break
 		}
 
+		if o.Namespace != "" {
+			val, err := RegexChecker(o.Namespace, res.NamespaceName)
+			if !val {
```

Comment:
```suggestion
			match, err := RegexChecker(o.Namespace, res.NamespaceName)
			if !match {
```
It returns whether it was Matched or not, so I think it would be better to use a variable name that conveys the same.